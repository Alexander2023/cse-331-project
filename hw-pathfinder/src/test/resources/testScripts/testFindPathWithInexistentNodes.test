# Tests that path finding works properly when asked to find path between inexistent nodes

CreateGraph g

FindPath g p1 p2