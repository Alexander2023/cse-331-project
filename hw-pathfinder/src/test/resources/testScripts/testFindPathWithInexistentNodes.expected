# Tests that path finding works properly when asked to find path between inexistent nodes

created graph g

unknown: p1
unknown: p2